src/Microsoft.Sbom.Api/Executors/Spdx3SerializationStrategy.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Api.Utils;
+using Microsoft.Sbom.Extensions;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+/// <summary>
+/// Serialization methods for SPDX 3.0.
+/// </summary>
+public class Spdx3SerializationStrategy : IJsonSerializationStrategy
```

Comment:
Let's make `JsonSerializationStrategyFactory`, `Spdx3SerializationStrategy`, `Spdx2SerializationStrategy`, and `IJsonSerializationStrategy` all `internal` unless we need them for unit tests. They shouldn't be part of the public API.
```suggestion
internal class Spdx3SerializationStrategy : IJsonSerializationStrategy
```